app/tests/communication/test_banner_integration.py

```diff
@@ -0,0 +1,56 @@
+from rest_framework import status
+
+import pytest
+
+from app.communication.factories.banner_factory import BannerFactory
+
+API_BANNER_URL = "/banners/"
+
+
+def _get_banner_url():
+    return f"{API_BANNER_URL}"
+
+
+def _get_visible_banner_url():
+    return f"{API_BANNER_URL}visible/"
```

Comment:
Veldig pirk, men disse kan fint være konstanter alle sammen:
```suggestion
API_VISIBLE_BANNER_URL = f"{API_BANNER_URL}visible/"
```

Reply:
burde det være det for alle nye testfiler? De fleste har et oppsett som dette. Enig med at `_get_banner_url` er litt ubrukelig. Har ingen problem med å endre de til bare konstanter.

Reply:
Jeg synes dette er bedre siden funksjonene ikke har noen effekt annet enn å returnere variablene. Vi bør egentlig ha alle samlet i en fil og ha et standard format.